### PR TITLE
Feature/185/column resources

### PR DESCRIPTION
**Pull request by ElenSuvarianTer**
# What does this PR do?

- Resources column added
- Added link to project in project name
- Tier link added in tier name

# Where should the reviewer start?
Activate Table view to see it

# How should this be manually tested?
Clicking on the added options

# Any background context you want to provide?

# What are the relevant tickets?

# Screenshots
Captura de pantalla 2023-07-25 a las 13 52 45

# Questions

# Checklist

- [x] I added the necessary documentation, if appropriate.
- [ ] I added tests to prove that my fix is effective or my feature works.
- [ ] I reviewed existing Pull Requests before submitting mine.

### CODE REVIEW

File: app/models/project.server.ts [hunk 1]
```diff
@@ -651,7 +652,7 @@ export async function searchProjects({
     projectIdsWhere = Prisma.sql`p.id IN (${Prisma.join(
       ids.map((val) => val.id)
     )})`;
-  }
+  }
```
Comment (jackbravo) [inline app/models/project.server.ts:655]: nitpick, but here it shows that there is extra whitespace at the end of this line :-p

File: app/routes/projects/index.tsx [hunk 2]
```diff
@@ -678,33 +678,33 @@ export default function Projects() {
...
-                        </TableCell>
-                        <TableCell>{item.tierName}</TableCell>
-                        <TableCell>{item.status}</TableCell>
-                        <TableCell>{item.projectMembers}</TableCell>
-                        <TableCell>{item.votesCount}</TableCell>
-                        <TableCell>{item.searchSkills}</TableCell>
-                      </TableRow>
+                      <TableCell><a href={`/projects/${item.id}`} style={{ color: 'inherit', textDecoration: 'none' }}>{item.name}</a></TableCell>
+                      <TableCell>
+                        <a href="https://wizeline.atlassian.net/wiki/spaces/wiki/pages/3075342381/Innovation+Tiers"
```
Comment (jackbravo) [inline app/routes/projects/index.tsx:696]: I'm thinking we could instead make the heading cell the link to what are innovation tiers, and make each project individual tier cell link to this page but filtered by that innovation tier. Similar to the innovation tier links inside a project view, like https://labs.wizeline.io/projects/ffc937dd-fbca-4855-9e2d-e895fc434332 that points to https://labs.wizeline.io/projects?tier=Tier%201

Same for project status and each skill. To link to the project list page filtered by status or skill.

**Review by jackbravo — CHANGES_REQUESTED** [review]

**Review by martinrobled0 — CHANGES_REQUESTED** [review]
same as Jack's comments jeje

File: app/models/project.server.ts (hunk 1 above)
Comment (ElenSuvarianTer) [reply app/models/project.server.ts]: Fixed

File: app/routes/projects/index.tsx (hunk 2 above)
Comment (ElenSuvarianTer) [reply app/routes/projects/index.tsx]: Fixed

**Review by martinrobled0 — APPROVED** [review]
LGTM

**Pull request merged**